{"uuid":"fe46365d-f44d-45c3-ab09-590e6286e632","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["245febfe-29b3-4b3f-a79c-acd397d589ce"],"befores":[],"afters":[{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687829869803,"stop":1687829869804},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687829869994,"stop":1687829869994},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687829870096,"stop":1687829870096},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687829870156,"stop":1687829870156},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687829870169,"stop":1687829870169}],"start":1687829869690,"stop":1687829870169}